
 
记录时间：2016-03-05 10:03:17,283 
线程ID:[14] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：LogInBranch
System.ServiceModel.CommunicationObjectFaultedException: 通信对象 System.ServiceModel.Channels.ServiceChannel 无法用于通信，因为其处于“出错”状态。

Server stack trace: 
   在 System.ServiceModel.Channels.CommunicationObject.Close(TimeSpan timeout)
   在 System.ServiceModel.Channels.CommunicationObject.Close()

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 System.ServiceModel.ICommunicationObject.Close()
   在 Kemai.MSG.Business.ClientLogin.LogInBranch()

 
记录时间：2016-03-05 11:01:54,641 
线程ID:[19] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 11:02:25,823 
线程ID:[19] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 11:02:33,391 
线程ID:[19] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 13:39:47,003 
线程ID:[19] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 13:42:45,936 
线程ID:[19] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 13:43:39,268 
线程ID:[19] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 13:44:36,068 
线程ID:[19] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 13:45:07,896 
线程ID:[24] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 13:46:25,780 
线程ID:[26] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 从数据类型 nvarchar 转换为 datetime 时出错。
   在 Kemai.MSG.DataAccess.ItemDal.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String ProductionType)
   在 Kemai.MSG.Business.MallLogic.RetailSettleOrder(MallOrder order, List`1 list, List`1 payOrder, String OperId, String CashNo)
ClientConnectionId:a5d3efe3-1bd5-4b06-b569-072430282141
Error Number:8114,State:5,Class:16

 
记录时间：2016-03-05 14:00:07,323 
线程ID:[9] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 找不到存储过程 'kmmicro_pr_check_vipcostorder'。
   在 Kemai.MSG.DataAccess.VipDal.CheckVipCostOrder(String vip_no, String sheetNo)
   在 Kemai.MSG.Business.MemberLogic.CheckVipCostOrder(String vip_no, String sheetNo)
ClientConnectionId:236e72f6-1cfe-4b3a-ad54-3f571213f4d5
Error Number:2812,State:62,Class:16

 
记录时间：2016-03-05 14:30:07,235 
线程ID:[9] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 找不到存储过程 'kmmicro_pr_check_vipcostorder'。
   在 Kemai.MSG.DataAccess.VipDal.CheckVipCostOrder(String vip_no, String sheetNo)
   在 Kemai.MSG.Business.MemberLogic.CheckVipCostOrder(String vip_no, String sheetNo)
ClientConnectionId:236e72f6-1cfe-4b3a-ad54-3f571213f4d5
Error Number:2812,State:62,Class:16

 
记录时间：2016-03-05 15:00:07,284 
线程ID:[9] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：System.Data.SqlClient.SqlException (0x80131904): 找不到存储过程 'kmmicro_pr_check_vipcostorder'。
   在 Kemai.MSG.DataAccess.VipDal.CheckVipCostOrder(String vip_no, String sheetNo)
   在 Kemai.MSG.Business.MemberLogic.CheckVipCostOrder(String vip_no, String sheetNo)
ClientConnectionId:236e72f6-1cfe-4b3a-ad54-3f571213f4d5
Error Number:2812,State:62,Class:16

 
记录时间：2016-03-05 15:24:08,870 
线程ID:[1] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：LogOutBranch
System.ServiceModel.CommunicationException: 服务器未提供有意义的回复；这可能是由协定不匹配、会话过早关闭或内部服务器错误引起的。

Server stack trace: 
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Kemai.O2O.Contract.IToOnlineService.LogonOut(String branchKey)
   在 Kemai.MSG.Business.ClientLogin.LogOutBranch()

 
记录时间：2016-03-05 17:20:11,042 
线程ID:[1] 
日志级别：ERROR 
出错类：RollingLogFileAppender property: [(null)] - 
错误描述：LogOutBranch
System.ServiceModel.CommunicationException: 服务器未提供有意义的回复；这可能是由协定不匹配、会话过早关闭或内部服务器错误引起的。

Server stack trace: 
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Kemai.O2O.Contract.IToOnlineService.LogonOut(String branchKey)
   在 Kemai.MSG.Business.ClientLogin.LogOutBranch()
